README.md

```diff
@@ -928,6 +930,114 @@ config.serializationConfig.portableFactories[1] = new PortableFactory();
 
 Note that the ID that is passed to the `SerializationConfig` is same as the `factoryId` that `Customer` object returns.
 
+
+### 4.2.1. Versioning for Portable Serialization
+
+More than one version of the same class may need to be serialized and deserialized. For example, a client may have an older version of a class and the member to which it is connected may have a newer version of the same class.
+
+Portable serialization supports versioning. It is a global versioning, meaning that all portable classes that are serialized through a member get the globally configured portable version.
+
+In NodeJs Client, you can achieve this by simply adding the `getClassIdd()` method to your class’s implementation of `Portable`, and setting the `ClassVersion` to be different than the default global version.
+
+
+A sample portable implementation of a `Employee2` class looks like the following:
+
+```javascript
+function Employee2(name,age,manager) {
```

Comment:
Can you change this sample with the python counter-part ? It would be good to have same examples across different client readmes.